tests/inputData.json.example

```diff
@@ -0,0 +1,45 @@
+{
+    "SDDC Manager": "sfo-vcf01.sfo.rainpole.io",
+    "SDDC Manager VM Name": "sfo-vcf01",
+    "User": "administrator@vsphere.local",
+    "Password": "VMw@re123!",
```

Comment:
```suggestion
    "Password": "VMware1!",
```

Reply:
@Ashwini-Ravi addressed this in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-certificate-management/commit/400b75366f04d6c47d1edc5acbd905f9219adead.